.eslintrc.js

```diff
@@ -66,6 +66,16 @@ module.exports = {
         '@typescript-eslint/naming-convention': 'warn',
       },
     },
+    // @metamask/keyring-utils
+    {
+      files: ['packages/keyring-utils/src/**/*.ts'],
+      extends: ['@metamask/eslint-config-typescript'],
+      parserOptions,
+      rules: {
+        // TODO: re-lint everything once the migration is done
```

Comment:
I noticed a some TODOs and FIXMEs. We should open a ticket to not lose track of this

Reply:
Those are here since the creation of the monorepo yes. I'll create a ticket to get back to this later on:
- https://github.com/MetaMask/accounts-planning/issues/756